fw/http_parser.c

```diff
@@ -4768,11 +4624,32 @@ Req_Method_1CharStep: __attribute__((cold))
 		__FSM_MOVE_nofixup_n(Req_MUSpace, 0);
 	}
 
+	__FSM_STATE(Req_UriRareForms, cold) {
+		/* TODO Support authority form as in RFC7230#section-5.3.3
+		 * when CONNECT method will be added */
```

Comment:
That's right. CONNECT method not needed for websockets. And maybe not needed for Tempesta FW at all, because it is not an explicit proxy. But i have seen in code mentions and TODOs of CONNECT:

```
root@devvps:~/tempesta# grep -r -I -w CONNECT
fw/t/unit/test_http_parser.c:   TEST_REQ_UNKNOWN(CONNECT);
fw/http_parser.c:        * DELETE, TRACE and CONNECT requests has no defined semantics  \
fw/http_parser.c:                * (Successful) response to a CONNECT request
fw/http_parser.c:        * in any 2xx (Successful) response to a CONNECT request.
fw/http_parser.c:                * when CONNECT method will be added */
fw/http_parser.c:                       /* TODO: Add (req == CONNECT && resp == 2xx) */
fw/http.h:/* TODO: When CONNECT will be added, add it to tfw_handle_validation_req()
fw/cache.c:     /* TODO: Add CONNECT */
```

Reply:
To be consistent with this intention i have added the comment for clarity and completeness. I'll change its content to reflect your note.